packages/trpc/server/routers/viewer/eventTypes.tsx

```diff
@@ -413,14 +413,10 @@ export const eventTypesRouter = router({
           id: userId,
         },
       },
+      ...((typeof rest?.locations === "undefined" || rest.locations?.length === 0) &&
```

Comment:
currently `location` is undefined while creating a event, so added that to the check as well